packages/core/src/sass/app/_common/layout/modal-root.scss

```diff
@@ -22,3 +23,7 @@
         z-index: 9999;
     }
 }
+
+.launch-modal-root:not(:empty) + .modal-root:not(:empty) {
```

Comment:
To hide next sibling (pop-up in modal root, if it's not empty) and show only one modal. When it'll be closed, show another